src/perform_step.jl

```diff
@@ -124,7 +124,7 @@ function OrdinaryDiffEq.perform_step!(integ, cache::EKCache, repeat_step=false)
 
     # Update the global diffusion MLE (if applicable)
     if !isdynamic(cache.diffusionmodel)
-        cache.global_diffusion = estimate_global_diffusion(cache.diffusionmodel, integ)
+        estimate_global_diffusion(cache.diffusionmodel, integ)
```

Comment:
This function should get an exclamation mark!

Reply:
And maybe even a completely different type signature, like `estimate_global_diffusion!(integ, cache)`?